src/drive/web/modules/move/MoveModal.jsx

```diff
@@ -90,10 +94,37 @@ class MoveModal extends React.Component {
     }
   }
 
-  moveEntry = (entryId, destinationId) => {
-    return this.context.client
-      .collection('io.cozy.files')
-      .updateFileMetadata(entryId, { dir_id: destinationId })
+  moveEntry = async (entryId, destinationId, entry) => {
```

Comment:
jsdoc ;)

Reply:
I will definitely when the code is OK :-)